client/src/pages/Home.tsx

```diff
@@ -15,7 +15,7 @@ export const Home = () => {
 
   const handleInputChange = async (query: string) => {
     try {
-      setResults(await fetchClient.getData<Course[]>(`/search?query=${query}`));
+      setResults((await fetchClient.getData<Course[]>(`/search?query=${query}`)).slice(0, 6));
```

Comment:
We don't need to slice courses here, we can limit the # of courses suggested on the backend, see https://github.com/terror/mcgill.gg/blob/master/crates/db/src/db.rs#L104

Reply:
dope, ill make a change to that method then. 10 items seem to overflow any screen thats not my 32inch monitor.

Reply:
hmm, what about adding a scroll overflow to the search results div then? 10 seems like a good amount to show, so we should just let users scroll through them if it overflows

Reply:
Sure, it was already scroll overflow, so I just reversed the array slice

Reply:
actually i just tried this out and the overflow seems fine, users can still scroll the page to see all of the results